### PR TITLE
document lxc task config option

### PR DESCRIPTION
**Pull request by h0tw1r3**
Document https://github.com/hashicorp/nomad-driver-lxc/pull/38

### CODE REVIEW

**Review by DerekStrickland — CHANGES_REQUESTED** [review]
Thanks for submitting this!  I have a quick request to make the md links consistent. Please see inline. You can just accept the suggestions and it will apply a patch.

File: website/content/plugins/drivers/community/lxc.mdx [hunk 1]
```diff
@@ -184,6 +186,7 @@ This driver supports CPU and memory isolation via the `lxc` library. Network
 isolation is not supported as of now.
 
 [lxc-create]: https://linuxcontainers.org/lxc/manpages/man1/lxc-create.1.html
+[lxc.conf]: https://linuxcontainers.org/lxc/manpages/man5/lxc.conf.5.html
```
[the remark this answers is not in the document]
Comment (h0tw1r3) [reply website/content/plugins/drivers/community/lxc.mdx]: done

File: website/content/plugins/drivers/community/lxc.mdx [hunk 2]
```diff
@@ -114,6 +114,8 @@ The `lxc` driver supports the following configuration in the job spec:
 
 - `gpg_key_id`: GPG key ID used for checking image signatures. Default is set by the underlying LXC library.
 
+- `config_file` - The `lxc` configuration file to load (commonly defaults to `/etc/lxc/default.conf`). See [`lxc.conf`][lxc.conf] for more details.
```
[the remark this answers is not in the document]
Comment (h0tw1r3) [reply website/content/plugins/drivers/community/lxc.mdx]: done

**Review by DerekStrickland — APPROVED** [review]
Looks good to me.  Tagging @lgfa29  for a second set of eyes.

**Review by tgross — COMMENTED** [review]
Note this blocked on merging https://github.com/hashicorp/nomad-driver-lxc/pull/38

**Comment by aimeeu** [comment]
Nomad 1.9 removes the deprecated LXC driver so I'm closing this PR.

**Pull request closed without merging**